source/includes/api/_layer.md

```diff
@@ -350,185 +524,307 @@ curl -X GET https://api.resourcewatch.org/v1/layer?includes=user
 
-Field             |              Description               |   Type |                                          Values | Required
------------------ | :------------------------------------: | -----: | ----------------------------------------------: | -------:
-name              |           Name of the layer            |   Text |                                        Any Text |      Yes
-description       |       Description of the dataset       |   Text |                                        Any text |       No
-application       | Application to which the layer belongs |  Array | gfw, forest-atlas, rw, prep, aqueduct, data4sdg |      Yes
-layerConfig       |          Custom configuration          | Object |                                    Valid object |       No
-legendConfig      |          Custom configuration          | Object |                                    Valid object |       No
-applicationConfig |          Custom configuration          | Object |                                    Valid object |       No
-staticImageConfig |          Custom configuration          | Object |                                    Valid object |       No
-iso               |  The isos to which the layer belongs   |  Array |                                         BRA, ES |       No
-dataset           |          UuId of the dataset           |   Text |                                 Uuid of Dataset |       No
-env               | The environment to which the layer belongs |   Text |                                    Any Text |       No
+- name
+- description 
+- application
 
-> To create a layer, you have to do a POST request with the following body:
+There's also a dependency on a dataset id, as it is required to build the POST URL. As noted on the [layer concept](#layer) documentation, a layer is meant to hold the rendering details of a layer, but not the actual data - that should be part of the dataset. While this is not enforced - it's up to your rendering tool to load the data, and it can do it from a RW API dataset of from anywhere else - it's common and best practice to have the data for a layer be sourced from the dataset that's associated with it.
+
+When a layer is created, a [vocabulary tag](#vocabulary-and-tags) for it is automatically created, associated with the dataset tag.
+
+The layer service was built to be very flexible, and not be restricted to specific layer rendering libraries or tools. This gives you the freedom to use virtually any rendering technology you want, but it also means you'll have to make additional decisions on how to structure your data into the different open format fields provided by a RW API layer. Further below we'll show you some examples of how existing applications use different rendering tools with the layer endpoints, to give you an idea on how you can structure your own data, and also as a way to help you get started creating your first layers for your own custom applications.
```

Comment:
Again, I would change this to "in the future" and I'll note in Jira that we need to change it when the tutorials are in place.

Reply:
:+1: